Formula/zabbix-cli.rb

```diff
@@ -2,11 +2,10 @@ class ZabbixCli < Formula
   include Language::Python::Virtualenv
 
   desc "CLI tool for interacting with Zabbix monitoring system"
-  homepage "https://github.com/usit-gd/zabbix-cli/"
-  url "https://github.com/usit-gd/zabbix-cli/archive/2.1.1.tar.gz"
```

Comment:
note, this did not get published into pypi index (also the URL got changed in the new release)